services/121-service/src/seed-data/program/program-cooperative-bank-of-oromia.json

```diff
@@ -0,0 +1,78 @@
+{
+  "published": true,
```

Comment:
We don't have published in our entity any more. It is still in most program jsons. I'll add a top-item taks.